build/stages/compliance.yml

```diff
@@ -1,79 +1,92 @@
-     displayName: Security & Analysis
-     pool: $(WindowsPoolName)
-     timeoutInMinutes: 60
-     cancelTimeoutInMinutes: 5 
-     steps:
-     - checkout: self
-       clean: true
-       submodules: recursive
-     - task: DownloadBuildArtifacts@0
-       displayName: Download Symbols
-       inputs:
+  - job: Compliance
+    displayName: Security & Analysis
+    pool: $(WindowsPoolName)
+    timeoutInMinutes: 60
+    cancelTimeoutInMinutes: 5
+    steps:
+    - checkout: self
+      clean: true
+      submodules: recursive
+
+    - task: DownloadBuildArtifacts@0
+      displayName: Download Symbols
+      inputs:
         artifactName: symbols
         downloadPath: '$(Build.ArtifactStagingDirectory)'
-     - task: AntiMalware@3
-       displayName: Run AntiMalware Scan
-       inputs:
-        FileDirPath: $(System.DefaultWorkingDirectory)
+
+    - task: AntiMalware@4
+      displayName: Run AntiMalware Scan
+      inputs:
+        InputType: 'Basic'
+        ScanType: 'CustomScan'
+        FileDirPath: '$(System.DefaultWorkingDirectory)'
         EnableServices: true
-       continueOnError: true
-       condition: succeededOrFailed()
-     - task: BinSkim@3
-       displayName: Run BinSkim Analysis
-       inputs:
+        TreatSignatureUpdateFailureAs: 'Warning'
+        SignatureFreshness: 'UpToDate'
+        TreatStaleSignatureAs: 'Error'
+
+    - task: BinSkim@4
+      displayName: Run BinSkim Analysis
+      inputs:
         InputType: Basic
-        AnalyzeTarget: '$(Build.ArtifactStagingDirectory)\symbols\*.dll'
+        Function: 'analyze'
+        TargetPattern: 'guardianGlob'
+        AnalyzeTargetGlob: '$(Build.ArtifactStagingDirectory)\Symbols\*.dll;$(Build.ArtifactStagingDirectory)\Symbols\*.exe'
         AnalyzeVerbose: true
-       continueOnError: true
-       condition: succeededOrFailed()
-     - template: security\credscan\v2.yml@templates # from xamarin/yaml-templates repository
-       parameters:
+
+    - template: security\credscan\v3.yml@templates # from xamarin/yaml-templates repository
+      parameters:
         suppressionsFile: $(System.DefaultWorkingDirectory)\build\CredScanSuppressions.json
-     - template: security\policheck\v1.yml@templates # from xamarin/yaml-templates repository
-       parameters:
+
+    - template: security\policheck\v2.yml@templates # from xamarin/yaml-templates repository
+      parameters:
         exclusionFile: $(System.DefaultWorkingDirectory)\build\PoliCheckExclusions.xml
-     - task: CodeInspector@2
-       displayName: Run Code Inspector Analysis
-       inputs:
-        ProductId: '$(System.TeamProjectId)'
-       continueOnError: true
-       condition: succeededOrFailed()
-     - template: security\publish-report\v1.yml@templates # from xamarin/yaml-templates repository
-       parameters:
+        pE: '1|2|3|4'
+
+    - task: CodeInspector@2
+      displayName: Run Code Inspector Analysis
+      inputs:
+      ProductId: '$(System.TeamProjectId)'
```

Comment:
ProductId is incorrectly indented and that breaks the pipeline (https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=6979387&view=results)